content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -43,11 +43,11 @@ The controllers inside the cloud controller manager include:
 
 ### Node controller
 
-The node controller is responsible for creating {{< glossary_tooltip text="Node" term_id="node" >}} objects
+The node controller is responsible for updating {{< glossary_tooltip text="Node" term_id="node" >}} objects
 when new servers are created in your cloud infrastructure. The node controller obtains information about the
 hosts running inside your tenancy with the cloud provider. The node controller performs the following functions:
 
-1. Initialize a Node object for each server that the controller discovers through the cloud provider API.
+1. Synchronize a Node object with the cloud provider's identifier for its server.
```

Comment:
What does `cloud provider's identifier` mean here? To me this isn't obvious and I know the tech so perhaps this needs to be reworded?

Reply:
i'm referring to the `providerID` is there a better way to state that?

Reply:
maybe i could say
> Synchronize a Node object with its server's unique identifier obtained from the cloud provider API.
?

Reply:
I guess I'm left wondering what is being synced though? We provider ID is set from kubelet right?

I was wondering if we should just remove the 1st point from this list 🤔

Reply:
> I guess I'm left wondering what is being synced though? We provider ID is set from kubelet right?

well yes, but also there is this https://github.com/kubernetes/cloud-provider/blob/master/controllers/node/node_controller.go#L477 , it does try to synchronize provider id in cases where it's not set. i guess that was all i was trying to capture here.

> I was wondering if we should just remove the 1st point from this list thinking

the thought had crossed my mind ;)

Reply:
i updated the language in an attempt to describe what is happening more accurately.